packages/arb-token-bridge-ui/src/generateOpenGraphImages.tsx

```diff
@@ -0,0 +1,345 @@
+        height: dimensions.height,
+        backgroundColor: 'black',
+        fontFamily: 'Unica77'
+      }}
+    >
+      <div
+        style={{
+          transform: 'rotate(180deg)',
+          mixBlendMode: 'overlay',
+          backgroundImage:
+            'url("https://arbitrum.io/background/grains_bottom.png")',
+          backgroundRepeat: 'no-repeat',
+          backgroundSize: 'cover',
+          width: '100%',
+          height: '100%',
+          opacity: 0.35
+        }}
+      />
+      <ArbitrumLogo
+        style={{
+          position: 'absolute',
+          top: '100px',
+          left: '10px'
+        }}
+        width={320}
+        height={345}
+      />
+      <span
+        style={{
+          position: 'absolute',
+          top: '425px',
+          left: '85px',
+          fontSize: '40px',
+          fontWeight: '500',
+          color: 'white',
+          letterSpacing: '4px'
+        }}
+      >
+        BRIDGE
+      </span>
+      <div
+        style={{
+          ...dimensions,
+          position: 'absolute',
+          top: 0,
+          left: '160px',
+          height: '100%',
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          justifyContent: 'center',
+          gap: '30px',
+          fontSize: '40px',
+          fontWeight: '400',
+          color: 'white'
+        }}
+      >
+        <ChainWrapper chain={from} direction="From" />
+        <img
+          src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACkAAABBCAYAAABMx43BAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAF3SURBVHgB7dmBcYIwFMbxh9cBOooj0A26AWzQEXSDdgPaSWwnqBvIBrpB+tKG9qpCXpIvAe/e/y6ndyD8AME7QwTKGPPE42B+sq8bWlIMejbX29ESYkhtpqspsRWlV3uWN5QYApk9RaJSJCpFolIkKkWiUiQqRaJSJCpFolIkKkWiUiQqRaJSJCpFolIkKkWiugnk3fCGZwnu6e9P+n1VVT0VzhkeeZwuDLyw4XE8m9ro3Id8G956pkg6EjQyF7QZFq4ndvDpgyKQ7oSM1drv5NT02prHzgjOaGzuINqJVZqVg0yVDSoA2mqL7MkfHCoE2nqL/CBZMGgA0PY+fOjVyPt3M4XeOJ6b5Oq+vh/m/Dxq+eWNZEWf0cAzuOfxwLbT7y9ObmgscGxjoZf+xbNOZyIuMQmOOgTq6xiwrgyYCYoHzgCNAxaEpgELQDHAjFAsMAM0DxAIzQsEQMsAE6BlgRHQeYAB0HmBAugygEOMaR3KduCxJVBfLnVlwvIHe58AAAAASUVORK5CYII="
+          alt="arrow"
+          width={41}
+          height={67}
+        />
+        <ChainWrapper chain={to} direction="To" />
+      </div>
+    </div>
+  )
+}
+
+async function getOrbitChainImage(orbitChain: Chain) {
+  const chainConfig = getBridgeUiConfigForChain(orbitChain)
+  const isSvg = chainConfig.network.logo.endsWith('.svg')
+  const logoFileBuffer = fs.readFileSync(`./public${chainConfig.network.logo}`)
+
+  console.log(`Generating image for ${orbitChain}`)
+
+  const imageContent = await (async function () {
+    if (isSvg) {
+      return await sharp(logoFileBuffer)
+        .resize({
+          width: 120,
+          height: 120,
+          fit: 'contain',
+          background: 'transparent'
+        })
+        .toBuffer()
+    }
+    return await sharp(logoFileBuffer).png().toBuffer()
+  })()
```

Comment:
NIT: I think we don't need IIFE here, we can simply use ternary

Reply:
good catch, i was trying things here but didn't revert